article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -342,16 +311,17 @@ object DotcomponentsDataModel {
       Configuration.rendering.sentryHost,
       Configuration.rendering.sentryPublicApiKey,
       switches,
-      linkedData,
+      linkedData = linkedData,
       Configuration.google.subscribeWithGoogleApiUrl,
       guardianBaseURL = Configuration.site.host,
       webURL = article.metadata.webUrl,
       shouldHideAds = article.content.shouldHideAdverts,
       hasStoryPackage = articlePage.related.hasStoryPackage,
       hasRelated = article.content.showInRelated,
       isCommentable = article.trail.isCommentable,
-      article.metadata.commercial,
-      article.content.starRating
+      commercialProperties = article.metadata.commercial,
+      starRating = article.content.starRating,
+      trailText = article.trail.fields.trailText.getOrElse("")
```

Comment:
I think you've added trailText as a `description` meta tag, what was the reason to do that vs `description` on the json+ld as described as optional field on https://developers.google.com/search/docs/data-types/article#amp

Reply:
So, Google say you should always add a description meta tag. It's unclear if that allows for SD, but it doesn't mention that - https://support.google.com/webmasters/answer/79812?hl=en - it seems like they require you to have a title meta tag and description for all articles. But it is definitely unclear.

See, e.g., https://support.google.com/webmasters/answer/35624?hl=en for an example of where they say `Make sure that every page on your site has a meta description.`